src/Stratis.Bitcoin.Features.PoA/PoANetwork.cs

```diff
@@ -64,7 +55,7 @@ public PoANetwork()
             // Configure federation public keys.
             // Keep in mind that order in which keys are added to this list is important
             // and should be the same for all nodes operating on this network.
-            this.FederationPublicKeys = new List<PubKey>()
+            var federationPublicKeys = new List<PubKey>()
```

Comment:
Can you bring this closer to the new PoAConsensusOptions()?
You can even bring it inside the instantiation and not use a variable.

Reply:
>Can you bring this closer to the new PoAConsensusOptions()?

done

>You can even bring it inside the instantiation and not use a variable.

static code analyzer use are using doesn't like it